src/ansys/dpf/post/__init__.py

```diff
@@ -21,14 +21,16 @@
 except ModuleNotFoundError:  # pragma: no cover
     import importlib_metadata
 
-# this must be after some ansys.dpf.post import
-__version__ = importlib_metadata.version("ansys-dpf-post")
-
+from ansys.dpf.post import tools
 from ansys.dpf.post.common import Grouping as grouping
 from ansys.dpf.post.dpf_path import create_path_on_coordinates
 from ansys.dpf.post.misc import Report
 from ansys.dpf.post.post_utility import load_solution, print_available_keywords
 
+# this must be after some ansys.dpf.post import
+__version__ = importlib_metadata.version("ansys-dpf-post")
```

Comment:
```suggestion
__version__ = importlib_metadata.version(__name__.replace(".", "-"))
```

Reply:
@GuillemBarroso not sure why, but this did not work when importing in the docstring tests

Reply:
Yep... We had a similar problem when not hardcoding the name in pydpf-core as well. No idea why tho.